jobs/bosh_dashboards/templates/bosh_deployments.json

```diff
@@ -227,7 +227,7 @@
         "multi": false,
         "name": "bosh_deployment",
         "options": [],
-        "query": "label_values(bosh_deployment_stemcell_info{environment=~\"$environment\",bosh_name=~\"$bosh_director\"}, bosh_deployment)",
+        "query": "label_values(bosh_deployment_stemcell_info{environment=~\"$environment\",bosh_name=~\"$bosh_director\",bosh_deployment!=\"bosh-health-check\"}, bosh_deployment)",
```

Comment:
To me it makes sense to show the `bosh-health-check` deployment at the `bosh deployments` dashboards, it will show you what releases and stemcells is using.